terraform/production/main.tf

```diff
@@ -66,30 +66,107 @@ module "application" {
   sidekiq_task_desired_count = 2
   sidekiq_task_memory        = 8192
 
-  ecr_repository_url      = "815624722760.dkr.ecr.eu-west-2.amazonaws.com/core"
+  ecr_repository_url = "815624722760.dkr.ecr.eu-west-2.amazonaws.com/core"
+
+  prefix                          = local.prefix
+  api_key_secret_arn              = module.application_secrets.api_key_secret_arn
+  app_host                        = local.app_host
+  app_task_desired_count          = local.app_task_desired_count
+  app_task_execution_role_arn     = module.application_roles.ecs_task_execution_role_arn

```

Comment:
Do we actually want to prefix `app_task_execution_role_arn` and `app_task_role_arn` with `app`, given they're being used by all 3 tasks?

Reply:
Good spot! I think these should be `ecs_task_`... instead of `app_task_` so have made that change 👍